src/rev_ai/models/insights/sentiment_analysis/sentiment_value.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""Enum for possible sentiments"""
+
+from enum import Enum
+
+
+class SentimentValue(Enum):
```

Comment:
should this be a union type?

Reply:
I don't believe union types don't exist in python